lua/tabnine/state.lua

```diff
@@ -1,4 +1,6 @@
 local uv = vim.loop
+local fn = vim.fn
+local utils = require("tabnine.utils")
```

Comment:
```suggestion
-local fn = vim.fn
-local utils = require("tabnine.utils")
```
both of these are unused.

edit: Created new comment to include GitHub suggestion

Reply:
```lua
local api = vim.api
local fn = vim.fn
```
These definitions in `lua/tabnine/keymaps.lua` line 2-3 are also unused, though they were added / made unused in a separate PR (70281f8c2e4b8f91a1c2f66f2a716959f026cd9b)